d2l-colors.html

```diff
@@ -44,6 +44,12 @@
 				--d2l-color-white: #fff;
 				--d2l-color-woolonardo: #f2f3f5;
 
+				/* alerts */
```

Comment:
I sort of have mixed feeling about it since all the colors are supposed to be from the shared palette here.  If they are included here I wouldn't call them out exclusively as alerts.  If they really are exclusively for alerts, I'd say they belong with the alerts repo.  I thought there may have been some effort made to build alerts web components (which is where I'd be keen to put these colors in such a case), but I don't know if that ever became a thing.